compiler/il/OMRNode.cpp

```diff
@@ -2829,21 +2829,12 @@ OMR::Node::isRematerializable(TR::Node *parent, bool onlyConsiderOpCode)
    return false;
    }
 
-
-
 bool
 OMR::Node::canEvaluate()
```

Comment:
Not sure what the purpose of this API was. It returns true on all paths. Sounds like it should be deprecated.

Reply:
Fixed in 67da1aa